data/presets/highway/cycleway/traffic_island.json

```diff
@@ -0,0 +1,26 @@
+{
+    "icon": "fas-biking",
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "cycleway": "traffic_island"
+    },
+    "addTags": {
+        "highway": "cycleway",
+        "cycleway": "traffic_island"
+    },
```

Comment:
I don't think this is needed, the `tags` is enough. We could move the `hw=cycleway` to `tags` but the again this should always match with `cycleway=*`.

```suggestion
```